modules/recruiting/storage/output.tf

```
@@ -0,0 +1,37 @@
+output "resource_group_name" {
+  value = azurerm_resource_group.tikjob_rg.name
+}
+
+output "resource_group_location" {
+  value = azurerm_resource_group.tikjob_rg.location
+}
+
+/* MySQL */
+output "mysql_fqdn" {
+  value = azurerm_mysql_server.tikjob_mysql.fqdn
+}
+
+output "mysql_connection_user" {
+  value = "${azurerm_mysql_server.tikjob_mysql.administrator_login}@${azurerm_mysql_server.tikjob_mysql.name}"
+}
+
+output "mysql_password" {
+  value = azurerm_mysql_server.tikjob_mysql.administrator_login_password
+}
+
+/* Storage */
+output "storage_account_name" {
+  value = azurerm_storage_account.tikjob_storage_account.name
+}
+
+output "storage_account_key" {
+  value = azurerm_storage_account.tikjob_storage_account.primary_access_key
```

Hide from logs (to only use directly with the cli):
```suggestion
  value = azurerm_storage_account.tikjob_storage_account.primary_access_key
  sensitive = true
```